contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeListItem/index.vue

```diff
@@ -2,196 +2,190 @@
 
   <VHover :disabled="copying">
     <template #default="{ hover }">
-      <ContextMenuCloak :disabled="contextMenuDisabled">
-        <template #default="contextMenuProps">
-          <DraggableHandle v-bind="draggableHandle">
-            <template #default>
-              <VListTile
-                v-if="node"
-                class="content-list-item pa-0"
-                :class="{
-                  'compact': isCompact,
-                  hover: hover && !copying,
-                  active: (active || hover) && !copying,
-                  disabled: copying,
-                  highlight,
-                }"
-                data-test="content-item"
-                @click="handleTileClick"
+      <DraggableHandle v-bind="draggableHandle">
```

Comment:
All these difs are just tab diffs since we are removing the <ContextMenuCloack> and <template> tags.